homeassistant/components/jewish_calendar/sensor.py

```diff
@@ -264,9 +272,11 @@ def extra_state_attributes(self) -> dict[str, str]:
         if self._state is None:
             return attrs
 
-        return attrs
+        return attrs  # type: ignore[unreachable]
```

Comment:
I would actually remove this entire property as it serves no purpose here.

Reply:
Please remove the whole function.